README.md

```diff
@@ -14,8 +14,13 @@ fanciful decoration.
 
 # example
 
+Example files: [example/parse.js](./example/parse.js) (CommonJS) / [example/parse.mjs](./example/parse.mjs) (ECMAScript)
+
 ``` js
-var argv = require('minimist')(process.argv.slice(2));
+const minimist = require('minimist'); // (for CommonJS)
+// import minimist from 'minimist'; // (for ECMAScript)
+
+const argv = minimist(process.argv.slice(2));
```

Comment:
```suggestion
// for CJS
const argv = require('minimist')(process.argv.slice(2));

// for ESM
// import minimist from 'minimist'; // (for ECMAScript)
// const argv = minimist(process.argv.slice(2));
```

to showcase the differences here

Reply:
Short version: ok, yes

Long version: I personally dislike putting code on the require line, but it was the previous usage and is a pattern that was popular. Yargs used to show this too. Ok.

Reply:
it's a reason CJS is better than ESM :-)